tests/framework/extensions/ingresses/ingresses.go

```diff
@@ -1,12 +1,25 @@
 package ingresses
 
 import (
+	"strings"
+	"time"
+
 	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	v1 "github.com/rancher/rancher/tests/framework/clients/rancher/v1"
 	"github.com/rancher/rancher/tests/framework/extensions/charts"
+	"github.com/rancher/rancher/tests/framework/extensions/workloads/pods"
+	"github.com/sirupsen/logrus"
+	networking "k8s.io/api/networking/v1"
+	kwait "k8s.io/apimachinery/pkg/util/wait"
 )
 
 const (
+	defaultNamespace = "default"
 	IngressSteveType = "networking.k8s.io.ingress"
+	ingressPath      = "/index.html"
+	pod              = "pod"
+	RKE2IngressNginx = "rke2-ingress-nginx"
```

Comment:
Can we modify this function so it's not so specific to rke2?

Reply:
Spoke offline, addressed this.